### DISCUSSION TITLE
[placeholder] vs [attr.placeholder]

### DISCUSSION THREAD

**Discussion by theorlovsky** [taiga-family/taiga-ui#2770] [Q&A] [answered]
In Taiga 3, the `[tuiTextfieldExampleText]` was replaced with native placeholders. I wonder why did you choose the longer syntax via attribute binding over the shorter property one?

```html
<input [attr.placeholder]="someValue">

<!-- these two look more consistent -->
<input [placeholder]="someValue">
<input placeholder="this is not binding">
```

**Comment by waterplea** [comment, marked as answer]
Seems like just a mistake on our part, these things should behave the same. Some attributes have corresponding properties and some might not, so I guess migration was just written with safety in mind.

**Reply by theorlovsky** [reply]
got it. thanks for the quick reply
